Traceback (most recent call last):
  File "/home/masheenist/source/lab4-lab4-hanscom-llosa/src/pyyc/compile.py", line 85, in <module>
    main()
  File "/home/masheenist/source/lab4-lab4-hanscom-llosa/src/pyyc/compile.py", line 29, in main
    explicate_ast = explicate.explicate(ast, {})
  File "/home/masheenist/source/lab4-lab4-hanscom-llosa/src/pyyc/explicate.py", line 43, in explicate
    return Module(None, explicate(ast.node, variables))
  File "/home/masheenist/source/lab4-lab4-hanscom-llosa/src/pyyc/explicate.py", line 47, in explicate
    newStmts.append(explicate(statement, variables))
  File "/home/masheenist/source/lab4-lab4-hanscom-llosa/src/pyyc/explicate.py", line 50, in explicate
    return Discard(CallFunc("print_any", [explicate(ast.nodes[0], variables)]))
  File "/home/masheenist/source/lab4-lab4-hanscom-llosa/src/pyyc/explicate.py", line 128, in explicate
    return IfExp(Let(x, explicate(ast.test, variables), getBool(x, variables)), explicate(ast.then, variables), explicate(ast.else_, variables))
  File "/home/masheenist/source/lab4-lab4-hanscom-llosa/src/pyyc/explicate.py", line 79, in explicate
    return Let(x, explicate(ast.left, variables), Let(y, explicate(ast.right, variables), ifElseChain(conditions,actions)))
  File "/home/masheenist/source/lab4-lab4-hanscom-llosa/src/pyyc/explicate.py", line 130, in explicate
    raise Exception("No AST match: " + str(ast))
Exception: No AST match: Sub((CallFunc(Name('input'), [], None, None), Const(7)))
